js/pages/trackers.es6.js

```diff
@@ -1,27 +1,72 @@
 const Parent = window.DDG.base.Page;
-const WhitelistView = require('./../views/whitelist.es6.js');
-const WhitelistModel = require('./../models/whitelist.es6.js');
-const whitelistTemplate = require('./../templates/whitelist.es6.js');
+const TrackerListView = require('./../views/trackerlist.es6.js');
+const TrackerListModel = require('./../models/trackerlist.es6.js');
+const TrackerListTemplate = require('./../templates/trackerlist.es6.js');
 
+const SiteView = require('./../views/site.es6.js');
+const SiteModel = require('./../models/site.es6.js');
+const SiteTemplate = require('./../templates/site.es6.js');
+
+const SearchView = require('./../views/search.es6.js');
+const SearchModel = require('./../models/search.es6.js');
+const SearchTemplate = require('./../templates/search.es6.js');
+
+const ItemMenuView = require('./../views/itemMenu.es6.js');
+const ItemMenuModel = require('./../models/itemMenu.es6.js');
+const ItemMenuTemplate = require('./../templates/itemMenu.es6.js');
```

Comment:
Let's declare templates like `itemMenuTemplate` vs `ItemMenuTemplate`. Generally uppercase first letter is used to declare constructors. Our templates are just plain old functions. It's pretty common in the SERP codebase that functions are all uppercase, but I feel that this is confusing/misleading.